### PR TITLE
Add io.github.leolost2605.gradebook

### PR DESCRIPTION
**Pull request by leolost2605**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.leolost2605.gradebook.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+app-id: io.github.leolost2605.gradebook
+
+runtime: io.elementary.Platform
+runtime-version: '7.1'
+sdk: io.elementary.Sdk
```
Comment (hfiguiere) [inline io.github.leolost2605.gradebook.yml:5]: there is no such runtime available on flathub.

Comment (leolost2605) [reply io.github.leolost2605.gradebook.yml]: Sorry didn't realize was just coming from elementary :)
I switched it to gnome

**Comment by hfiguiere** [comment]
bot, build io.github.leolost2605.gradebook

File: io.github.leolost2605.gradebook.yml (hunk 1 above)
Comment (hfiguiere) [reply io.github.leolost2605.gradebook.yml]: tbf I just wish the Elementary people submitted their runtime here.

**Pull request closed without merging**